packages/terra-tabs/src/orion-fusion-theme/Tab.module.scss

```diff
@@ -1,6 +1,13 @@
 @import '~terra-mixins/lib/Mixins';
 
 :local {
+  .button:focus {
+    &[data-terra-tabs-show-focus-styles='true'] {
+      outline: var(--terra-tabs-add-button-focus-outline, 2px dashed #000);
+      outline-offset: var(--terra-tabs-add-button-focus-outline-offset, -2px);
+    }
+  }
```

Comment:
This is not valid for theme files Ideally you should just have variables and values like the other styles in this file . Please update the same for clinical-lowlight-theme as well

Reply:
Added https://github.com/cerner/terra-framework/commit/628052153b88461ef88733be6b05c1fb519e3f41